File: Level1/문자열을 정수로 바꾸기.py
#문자열을 정수로 바꾸기
def solution(s):
    answer = 0
    if s[0]=='-':
        answer+=int(s[1::])*-1
    else:
        answer+=int(s[1::])
    return answer
s='-1234'
print(solution(s))

File: Level1/문자열을 정수로 바꾸기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(string s) {
    int answer = 0;
    string tmp = "";
    if (s[0] == '-') {
        for (int i = 1; i < s.length(); i++) {
            tmp += s[i];
        }
        answer = -atoi(tmp.c_str());
    }
    else {
        for (int i = 0; i < s.length(); i++) {
            tmp += s[i];
        }
        answer = atoi(tmp.c_str());
    }
    return answer;
}